interop-testing/src/main/java/io/grpc/testing/integration/StressTestClient.java

```diff
@@ -186,6 +187,11 @@ void parseArgs(String[] args) {
       }
     }
 
+    if (!serverAddresses.isEmpty()) {
```

Comment:
`if (!usage && !serverAddresses.isEmpty())` 
because `usage` can be turned to true in the for-loop.